src/inc/til/bitmap.h

```diff
@@ -148,10 +148,14 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
         bitmap(til::size sz, bool fill) :
             _sz(sz),
             _rc(sz),
-            _bits(sz.area(), fill),
+            _bits(_sz.area()),
             _dirty(fill ? sz : til::rectangle{}),
             _runs{}
         {
+            if (fill)

```

Comment:
it looked to me like the ctor for dynamic bitset supported this still

Reply:
nnnnnnn I had that and it wasn't working right. It took the long long value of the bool as 1 and splatted that in a block instead of filling the whole block with bits.